kubernetes/helm_charts/networkconfig/values.j2

```diff
@@ -6,6 +6,7 @@ domain_name: {{ domain_name }}
 cloud_storage_url: {{ cloud_storage_url }}
 swarm_kibana_ip: {{groups['swarm-dashboard'][0]}}
 plugin_container_name: {{plugin_container_name}}
-offline_installer_container_name: {{ namespace }}-offlineinstaller
+offline_installer_container_name: {{ offline_installer_container_name }}
```

Comment:
Update on both side to ```desktop_container_name```. If this variable is not specific to kube and is being used in nginx also, then don't change the RHS